lib/utils/async.html

```diff
@@ -78,7 +78,6 @@
       /**
        * Enqueues a function called in the next task.
        *
-       * @function
```

Comment:
I wonder why we had these `@function` annotations... @justinfagnani do you know if these are important?

Reply:
some seemed to have it and some didn't. those which did would produce duplicate type declarations for some reason... so either we shouldn't have `@function`, or the generator is having trouble with it.

Reply:
Seems like a bug in polymer-analyzer.

Reply:
Not sure why it was annotated that way. I wonder if it had to do with the container being a namespace. It'd be good to run these annotation-changing PRs through doc generation.